build/yamls/antrea.yml

```diff
@@ -313,6 +318,22 @@ spec:
           name: antrea-config-fh9t4g64dc
         name: antrea-config
 ---
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  labels:
+    app: antrea
+  name: v1.antctl.antrea.io
+spec:
+  group: antctl.antrea.io
```

Comment:
The group here should be antctl.antrea.tanzu.vmware.com

Reply:
Changed.